.env.local.sample

```diff
@@ -8,3 +8,8 @@ SLACK_API_SECRET=
 SLACK_REDIRECT_URI=https://mojotech-helios2.herokuapp.com/web_hooks/create_slack_auth
 SLACK_VERIFICATION_TOKEN=
 SLACK_WEBHOOK_URL=
+GITHUB_WEBHOOK_SECRET=
```

Comment:
Do you know where we can find this on GitHub or how we add this to the project?

Reply:
Is the term 'WEBHOOK' similar to http callback?

Reply:
In github project settings (that you own or have admin access):
![Screenshot from 2021-05-25 07-46-25](https://user-images.githubusercontent.com/524850/119492758-717b0e00-bd2d-11eb-9008-a84e55f33dd9.png)